x/globalfee/README.md

```diff
@@ -2,3 +2,5 @@
 
 Global fee module is copied from the [tgrade](https://github.com/confio/tgrade) respository
```

Comment:
The Global fee module was supplied by the great folks at [TGrade](https://github.com/confio/tgrade).